frontend/app/src/components/home/addgroup/NewGroup.svelte

```diff
@@ -39,12 +39,7 @@
     let step = 0;
     let actualWidth = 0;
     let detailsValid = true;
-    let originalGroup = {
-        ...candidateGroup,
-        rules: { ...candidateGroup.rules },
-        permissions: { ...candidateGroup.permissions },
-        gate: { ...candidateGroup.gate },
-    };
+    let originalGroup = structuredClone(candidateGroup);
```

Comment:
Note: we really need to remember to test anything that uses `structuredClone` in safari (desktop and mobile)

Reply:
ok